x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/es_index/es_index.tsx

```diff
@@ -68,6 +70,32 @@ export function getActionType(): ActionTypeModel<EsIndexConfig, unknown, IndexAc
           )
         );
       }
+      if (actionParams.indexOverride) {
+        if (!actionParams.indexOverride.startsWith(ALERT_HISTORY_PREFIX)) {
+          errors.indexOverride.push(
+            i18n.translate(
+              'xpack.triggersActionsUI.components.builtinActionTypes.error.badIndexOverrideValue',
+              {
+                defaultMessage: 'Alert history index must begin with "{alertHistoryPrefix}".',
+                values: { alertHistoryPrefix: ALERT_HISTORY_PREFIX },
+              }
+            )
+          );
+        }
+
+        const indexSuffix = actionParams.indexOverride.replace(ALERT_HISTORY_PREFIX, '');
+        if (indexSuffix.length === 0) {
+          errors.indexOverride.push(
+            i18n.translate(
+              'xpack.triggersActionsUI.components.builtinActionTypes.error.badIndexOverrideSuffix',
+              {
+                defaultMessage: 'Alert history index must contain valid suffix',
```

Comment:
Ending period?

Reply:
Updated in https://github.com/elastic/kibana/pull/94909/commits/f3fea184ebc1e3e02f57b085216f9eab918bae29